warehouse/models/mart/gtfs/fct_daily_scheduled_stops.sql

```diff
@@ -0,0 +1,76 @@
+{{ config(materialized='table') }}
+
+WITH fct_daily_scheduled_trips AS (
+
+    SELECT
+
+        service_date,
+        feed_key,
+        service_id,
+        trip_id,
+        route_id,
+        route_type
+
+    FROM {{ ref('fct_daily_scheduled_trips') }}
+),
+
+dim_stop_times AS (
+    SELECT *
+    FROM {{ ref('dim_stop_times') }}
+),
+
+dim_stops AS (
+    SELECT *
+    FROM {{ ref('dim_stops') }}
+),
+
+dim_routes AS (
+    SELECT *
+    FROM {{ ref('dim_routes') }}
+),
+
+
+stops_by_day AS (
+
```

Comment:
Ok sorry I missed this before, I think that we may want to join with `dim_stops` here and group by `stop_key` rather than just by `stop_id` 🤔 Because that way people can do a versioned join if they need to? I kind of want @atvaccaro's opinion here

Reply:
Per discussion we will add it later, not here